hosts/cassini/iss.py

```diff
@@ -417,52 +418,52 @@ def reset():
 ################################################################################
 # UNIT TESTS
 ################################################################################
-
-import unittest
-import os.path
-
-from oops.unittester_support            import TESTDATA_PARENT_DIRECTORY
-from oops.backplane.exercise_backplanes import exercise_backplanes
-from oops.backplane.unittester_support  import Backplane_Settings
-
-
-#*******************************************************************************
-class Test_Cassini_ISS(unittest.TestCase):
-
-    def runTest(self):
-
-        from oops.unittester_support import TESTDATA_PARENT_DIRECTORY
-
-        snapshots = from_index(os.path.join(TESTDATA_PARENT_DIRECTORY,
-                                            'cassini/ISS/index.lbl'))
-        snapshot = from_file(os.path.join(TESTDATA_PARENT_DIRECTORY,
-                                          'cassini/ISS/W1575634136_1.IMG'))
-        snapshot3940 = snapshots[3940]  #should be same as snapshot
-
-        self.assertTrue(abs(snapshot.time[0] - snapshot3940.time[0]) < 1.e-3)
-        self.assertTrue(abs(snapshot.time[1] - snapshot3940.time[1]) < 1.e-3)
-
-
-#*******************************************************************************
-class Test_Cassini_ISS_Backplane_Exercises(unittest.TestCase):
-
-    #===========================================================================
-    def runTest(self):
-
-        if Backplane_Settings.NO_EXERCISES:
-            self.skipTest('')
-
-        root = os.path.join(TESTDATA_PARENT_DIRECTORY, 'cassini/ISS')
-        file = os.path.join(root, 'N1460072401_1.IMG')
-        obs = from_file(file)
-        exercise_backplanes(obs, use_inventory=True, inventory_border=4,
-                                 planet_key='SATURN')
-
-
-############################################
-from oops.backplane.unittester_support import backplane_unittester_args
-
-if __name__ == '__main__':
-    backplane_unittester_args()
-    unittest.main(verbosity=2)
+#
```

Comment:
I talked to Joe about this. Right now, there are no existing unit tests for numerous host modules, including cassini.iss. There is an open task to create them, which Joe has assigned to himself. I didn't want to delete whatever skeletal unit tests are there, because that info will help Joe to create the new tests based on gold_master.py. The case of Voyager ISS is a good example, where whatever unit test was ever there has failed because of the missing imports. I have now commented that code out as well, and will check it in shortly.